rules/windows/registry/registry_set/registry_set_asep_reg_keys_modification_office.yml

```
@@ -62,6 +62,7 @@ detection:
             - '\Outlook\Addins\UCAddin.LyncAddin.1'
             - '\Outlook\Addins\UCAddin.UCAddin.1'
             - '\Outlook\Addins\UmOutlookAddin.FormRegionAddin\'
+            - 'AddinTakeNotesService\FriendlyName'
```

```suggestion
            - '\AddinTakeNotesService\FriendlyName\'
```
if possible add path separators